### ISSUE TITLE
Buildroot image running sucessfully on Rpi Board

### ISSUE THREAD

**Issue by swapnilghonge** [cu-ecen-aeld/final-project-swapnilghonge#7]
**DoD:** 
Implementing the image of Buildroot inside Raspberry Pi 3b+ hardware

**Tasks:**
- Setup the Buildroot for Raspberry Pi by the following the steps from [](https://github.com/cu-ecen-aeld/buildroot-assignments-base/wiki/Raspberry-Pi-Hardware-Support)
- Copy the build image in SD card
- Boot the Raspberry to view buildroot login.

**Blockers:**
None

**Comment by swapnilghonge** [comment]
I used the steps to defined in buildroot for from raspberry Pi Hardware build Page.
To build buildroot for Rpi, I used my assignment-5 repository which has buildroot.  I cloned my Assignment -5 repository.I merged the my repo with hardware Rpi as defined in steps. I faced merge related issues for branch I was working on. I resolved them all .
Finally I gave a build for running buildroot for Rpi. Below is the screenshot for build completed.

![201537942-5e635401-633c-4b4c-bdc9-68fccedba22c](https://user-images.githubusercontent.com/36632481/201546364-df1bd80b-b5f4-4bf4-b4ff-86f8fc4d6ece.png)
